### DISCUSSION TITLE
Add support to width request/height request to Popup (instead of size or as addition to size)

### DISCUSSION THREAD

**Discussion by marekm294** [CommunityToolkit/Maui#1014] [New Feature Discussions]
It would nice to have opportunity to set only on of size properties and the other let adjust automatically. Controls in MAUI have HeightRequest and WidthRequest properties so it would be consistent.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
